src/generate.jl

```diff
@@ -51,24 +47,24 @@ function project(ctx::Context, pkg::String, dir::String)
     authors = ["$name " * (email === nothing ? "" : "<$email>")]
 
     uuid = UUIDs.uuid4()
-    genfile(ctx, pkg, dir, "Project.toml") do io
+    genfile(io, dir, "Project.toml") do file_io
         toml = Dict{String,Any}("authors" => authors,
                                 "name" => pkg,
                                 "uuid" => string(uuid),
                                 "version" => "0.1.0",
                                 )
-        TOML.print(io, toml, sorted=true, by=key -> (Types.project_key_order(key), key))
+        TOML.print(file_io, toml, sorted=true, by=key -> (Types.project_key_order(key), key))
     end
     return uuid
 end
 
-function entrypoint(ctx::Context, pkg::String, dir)
-    genfile(ctx, pkg, dir, "src/$pkg.jl") do io
-        print(io,
+function entrypoint(io::IO, pkg::AbstractString, dir)
+    genfile(io, dir, "src/$pkg.jl") do file_io
+        print(file_io,
            """
             module $pkg
 
-            greet() = print("Hello World!")
```

Comment:
Why change this? I like having some function in here that does something.

Reply:
Just because of [this comment](https://github.com/JuliaLang/Pkg.jl/pull/2492#issuecomment-816060851). Personally, I always found `greet() = print("Hello World!")` a bit weird, but I have no strong preference one way or the other.

Reply:
this function should not remove, there is nothing weird.

Reply:
Ok, I reverted it

Reply:
boo 😔😔😔😔😔

Reply:
Can't satisfy everyone 😄. We an do an emoji vote to decide:

* 👍 keep `greet() = print("Hello World!")`
* 👎 delete it

Reply:
put it in the main issue this is buried